src/components/calcite-flow/calcite-flow.tsx

```diff
@@ -26,6 +26,34 @@ export class CalciteFlow {
    */
   @Prop({ reflect: true }) theme: CalciteTheme;
 
+  // --------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  // --------------------------------------------------------------------------
+
+  /**
+   * Removes the currently active `calcite-flow-item`.
+   */
+  @Method()
+  async back(): Promise<HTMLCalciteFlowItemElement> {
+    const lastItem = this.el.querySelector(
+      "calcite-flow-item:last-child"
+    ) as HTMLCalciteFlowItemElement;
+
+    if (!lastItem) {
+      return;
```

Comment:
probably should return `Promise.resolve();` or `Promise.resolve(null);` since that's the expected return type.

Reply:
This is an async function so a promise is resolved. Are you suggesting to return null instead of undefined?

Reply:
@pr3tori4n ^

Reply:
> This is an async function so a promise is resolved. 

Wasn't sure about this, so yea a new promise wrapper is redundant and not needed.